system-tests/e2e-transfer-test/runner/src/test/java/org/eclipse/edc/test/e2e/TransferPullEndToEndTest.java

```diff
@@ -340,6 +316,68 @@ private HttpResponse cacheEdr(HttpRequest request, Map<String, TransferProcessSt
             }
         }
 
+
+        private String start_httpPull_dataTransfer(){
```

Comment:
as the previous review: change name format, this is not rust :)

plus, this method is not doing what is stating in the name: is not starting anything, just creating resources